src/datarecord.cpp

```diff
@@ -233,26 +230,22 @@ void datarecord::steady_bolus(odeproblem* prob, LSODA& solver) {
   }
   
   prob->rate_reset();
-  double N_SS = prob->ss_n;  
-  double CRIT_DIFF_SS = prob->ss_tol;
+  bool warn = !prob->ss_fixed;
```

Comment:
why no warning with `ss_fixed=true`?

Reply:
- There might be cases where the system won't go to steady state
- If the user gets into this situation, they will get warned
- `ss_fixed` is a way to let the user say "just give 100 doses and call that steady state" and 
the warning goes away at this point